modules/apps/static/portal-dependency-manager-component-executor-factory/portal-dependency-manager-component-executor-factory/src/main/java/com/liferay/portal/dependency/manager/component/executor/factory/internal/activator/ComponentExecutorFactoryBundleActivator.java

```diff
@@ -40,17 +43,29 @@
 
 	@Override
 	public void start(BundleContext bundleContext) {
-		boolean threadPoolEnabled = GetterUtil.getBoolean(
-			bundleContext.getProperty("dependency.manager.thread.pool.enabled"),
-			true);
+		boolean threadPoolEnabled = true;
+
+		long syncTimeout = 60;
+
+		_propsServiceReference = bundleContext.getServiceReference(Props.class);
+
+		if (_propsServiceReference != null) {
```

Comment:
@W2jx We know for sure that there is already a `Props` OSGi service at this point. If it doesn't exist it means bigger problem in our system. So I don't think the null protections here are needed. Or you can just use portal-kernel's `PropsUtil`.